src/pages/GamePage.jsx

```diff
@@ -26,11 +26,11 @@ const GamePage = () => {
     } else {
       event.preventDefault();
       setErrorHandler(false);
-      selectedArray.forEach(checkSelected);
+      checkedValue = checkingPreviousNumber(selectedArray, userNumber);
```

Comment:
tak jak mówiliśmy, możesz to wrzucić do poniższego `if` be` zapisywania do zmiennej.
To już nie jest straszna rzecz, ale żeby bylo ladniej to lepiej to poprawic